gcp/appengine/frontend_handlers.py

```diff
@@ -190,6 +191,28 @@ def add_source_info(bug, response):
   response['source_link'] = response['source']
 
 
+def add_related_aliases(bug: osv.Bug, response):
+  """Add links to other osv entries that's related through aliases"""
+  # Add links to other entires if they exist
+  aliases = []
+  for alias in bug.aliases:
+    result = bug.get_by_id(alias)
+    if result:
+      aliases.append({'alias_id': alias, 'exists': True})
+    else:
+      aliases.append({'alias_id': alias, 'exists': False})
+  response['aliases'] = aliases
+
+  # Add links to other entries if they have this element as an alias
+  if bug.aliases:
+    query = osv.Bug.query(
+        osv.Bug.aliases.IN(bug.aliases), osv.Bug.aliases == bug.db_id)
+  else:
+    query = osv.Bug.query(osv.Bug.aliases == bug.db_id)
+
+  response['alias_referenced'] = [x.id() for x in query if x.id() != bug.id()]
```

Comment:
Can this result in duplicates in our UI? If we have an entry in aliases, and then a reverse link here?

Reply:
Good point, added an additional filter to filter out entries that is already in aliases